pom.xml

```diff
@@ -502,6 +502,7 @@
                             -Xms${air.test.jvmsize}
                             -XX:OnOutOfMemoryError="kill -9 %p"
                             -XX:+HeapDumpOnOutOfMemoryError
+                            --add-modules java.base
```

Comment:
Is this in a java 9-only profile? Otherwise, it will fail with Java 8, no?

Reply:
Good catch, let me figure out how to do that.